src/workerd/api/html-rewriter.h

```diff
@@ -157,6 +157,23 @@ struct ContentOptions {
 
 class Rewriter;
 
+// Each Token subclass has an inner BaseImpl subclass which holds a reference
+// to the rewriter, and the actual underlying lol-html C API handle for the token.
+template <typename CType>
+struct BaseImpl {
```

Comment:
Nit: BaseImpl is quite a generic name to have in the workerd::api namespace. Can we nest this inside of HTMLRewriter::Token and call it ImplBase? (We typically name base classes FooBase, where Foo is our intended derived class name -- which in this case happens to be Impl, which is why I think that order makes more sense.)